loopchain/blockchain/blocks/v1_0/block_verifier.py

```diff
@@ -1,25 +1,137 @@
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Sequence
 
 from lft.consensus.messages.data import DataVerifier
 
-from loopchain.blockchain.transactions import TransactionVersioner
+from loopchain import configure_default as conf
+from loopchain import utils
+from loopchain.blockchain import ExternalAddress
```

Comment:
I think this line could raise import error, because module `blockchain` does not import module `blockchain.types` (Indirect import)

Possible solutions:
- `from loopchain.blockchain.types import ExternalAddress`
- import blockchain.types at `blockchain.__init__.py`
- Leave it as it is.

Reply:
Yes, I guess indirect imports should be refactoring all parts of this project.

Reply:
> Resolve the conversation if you want.